proof/refine/ARM/Finalise_R.thy

```diff
@@ -4368,62 +4489,388 @@ global_interpretation delete_one
    apply auto
   done
 
+(*FIXME RT: move*)
+lemma corres_split_new:
```

Comment:
This lemma is temporary and can be removed after https://github.com/seL4/l4v/pull/218 is merged into master and then rt.